src/main/resources/JavaAnnotationfree/pojo.mustache

```diff
@@ -103,15 +103,16 @@ public class {{classname}}{{#parent}} extends {{parent}}{{/parent}}{{#interfaces
 {{/vars}}
     }
 
-    public void mergeFrom({{classname}} target) {
+    public {{classname}} copyTo({{classname}} target) {
```

Comment:
I think the parameter type and return type should instead be `<T extends {{class name}}>`